iwyu_tool_test.py

```diff
@@ -363,6 +363,28 @@ def test_extra_args_with_sep(self):
         self.assertEqual(['arg1', '--', 'another_arg1'],
                          self.main.call_args['extra_args'])
 
+    def test_jobs_normal(self):
+        """ Nonzero values for -j are forwarded as they are. """
+        argv = ['iwyu_tool.py', '-p', '.', '-j', '4']
+        iwyu_tool._bootstrap(argv)
+        self.assertEqual(4, self.main.call_args['jobs'])
+
+
+    def test_jobs_auto(self):
+        """ -j=0 will not hang the program. """
+        argv = ['iwyu_tool.py', '-p', '.', '-j', '0']
+        iwyu_tool._bootstrap(argv)
+        self.assertNotEqual(0, self.main.call_args['jobs'])
+        self.assertNotEqual(None, self.main.call_args['jobs'])
+
+    def test_jobs_novalue(self):
+        """ It is allowed to use -j without a value """
+        argv = ['iwyu_tool.py', '-j', '-p', '.']
+        iwyu_tool._bootstrap(argv)
+        self.assertNotEqual(0, self.main.call_args['jobs'])
+        self.assertNotEqual(None, self.main.call_args['jobs'])
+
+
```

Comment:
I'd like to see a testcase here for a command-line without `-j` -- I think that will currently evaluate to `jobs == 0`.